scripts/sort_contibutors/main.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import json
+import os
+from dotenv import load_dotenv
+
+# It's just for debugging purpose
+
+
+def print_list(user_list: dict, label="") -> None:
+
+    print(label)
+    for value in user_list:
+        print(value['name'], " ", value['username'], ' ', value['ranking'])
+    print("=============================================")
+
```

Comment:
looking forward to the html and makrdown printing :)

Reply:
I'm working on it.